.github/workflows/team-channels.json

```diff
@@ -4,11 +4,11 @@
     "cross-chain-team": "eng-cross-chain-prs",
     "crypto-team": "eng-crypto-mr",
     "DRE": "eng-release-bots",
-    "execution": "eng-execution-mrs",
+    "execution": "eng-dsm-prs",
```

Comment:
We can probably also drop this entirely. After this PR is merged, review will only be requested from `team-dsm` anyways. Any objections?
```suggestion
```